### PR TITLE
Added audit ms to Postfilter

### PR DESCRIPTION
**Pull request by ayancovich**

### CODE REVIEW

File: src/main/java/com/endava/controller/RefreshController.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void refreshServices() {
 		.forEach(x -> {
 			x.getInstances().stream().forEach(i -> {
 				List<String> list_ms = Arrays.asList("http://", String.valueOf(i.getIPAddr()), ":",
-						String.valueOf(i.getPort()), "/refresh");
+						String.valueOf(i.getPort()), "/actuator/refresh");
```
Comment (nestorreloaded) [inline src/main/java/com/endava/controller/RefreshController.java]: This should be a constant

File: src/main/java/com/endava/filters/PostFilter.java [hunk 2]
```diff
@@ -1,28 +1,101 @@
...
-		return false;
+		return true;
 	}
 
+	@Override
 	public Object run() throws ZuulException {
-		// TODO Auto-generated method stub
+		auditCalls();
+
 		return null;
```
Comment (nestorreloaded) [inline src/main/java/com/endava/filters/PostFilter.java:62]: Please don't return null

File: src/main/java/com/endava/filters/PostFilter.java [hunk 3]
```diff
@@ -1,28 +1,101 @@
...
+				request_body = request.getReader().lines().collect(Collectors.joining());
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+			data.put(SecurityConstants.AUDIT_REQUEST_BODY, request_body);
+			try (final InputStream responseDataStream = ctx.getResponseDataStream()) {
+				response_body = CharStreams.toString(new InputStreamReader(responseDataStream, "UTF-8"));
+				ctx.setResponseBody(response_body);
+			} catch (IOException e) {
+				// logger.warn("Error reading body", e);
```
Comment (nestorreloaded) [inline src/main/java/com/endava/filters/PostFilter.java]: We should do something in the exception, at least log the error

File: src/main/java/com/endava/filters/PostFilter.java [hunk 4]
```diff
@@ -1,28 +1,101 @@
...
+			String request_body = "";
+			String response_body = "";
+			try {
+				request_body = request.getReader().lines().collect(Collectors.joining());
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+			data.put(SecurityConstants.AUDIT_REQUEST_BODY, request_body);
+			try (final InputStream responseDataStream = ctx.getResponseDataStream()) {
+				response_body = CharStreams.toString(new InputStreamReader(responseDataStream, "UTF-8"));
```
Comment (nestorreloaded) [inline src/main/java/com/endava/filters/PostFilter.java]: Use constants

File: src/main/java/com/endava/filters/PreFilter.java [hunk 5]
```diff
@@ -65,7 +71,7 @@ public Object run() throws ZuulException {
 			String URI = request.getRequestURI();
 
 			JSONObject data = new JSONObject();
-			data.put("token", token); 
+			data.put("token", token);
```
Comment (nestorreloaded) [inline src/main/java/com/endava/filters/PreFilter.java]: Use constants

**Review by nestorreloaded — CHANGES_REQUESTED** [review]
Please see the notes

File: src/main/java/com/endava/controller/RefreshController.java (hunk 1 above)
Comment (ayancovich) [reply src/main/java/com/endava/controller/RefreshController.java]: Done

**Review by nestorreloaded — APPROVED** [review]

**Review by exe135 — APPROVED** [review]

**Pull request merged**